anta/tests/stun.py

```diff
@@ -74,42 +70,24 @@ def test(self) -> None:
         # Iterate over each command output and corresponding client input
         for command, client_input in zip(self.instance_commands, self.inputs.stun_clients):
             bindings = command.json_output["bindings"]
-            source_address = str(command.params.source_address)
-            source_port = command.params.source_port
+            input_public_address = client_input.public_address
+            input_public_port = client_input.public_port
 
             # If no bindings are found for the STUN client, mark the test as a failure and continue with the next client
             if not bindings:
-                self.result.is_failure(f"STUN client transaction for source `{source_address}:{source_port}` is not found.")
+                self.result.is_failure(f"{client_input} - STUN client transaction not found.")
```

Comment:
This was an error from the first test
```suggestion
                self.result.is_failure(f"{client_input} - STUN client translation not found.")
```

```
site3-wan1#show stun client ?
  counters      Display STUN client counters
  translations  Show STUN client translations
```

Reply:
Updated